helm_deploy/values-preprod.yaml

```diff
@@ -6,11 +6,11 @@ generic-service:
 
   resources:
     requests:
-      cpu: 80m
+      cpu: 150m
       memory: 1000Mi
     limits:
       cpu: 2000m
-      memory: 2000Mi
+      memory: 2500Mi
```

Comment:
I noticed a couple of out of memory crashes recently so upped this a little. Also cpu requests level dictates when the auto-scalar fires up (which we probably want to drop entirely, but for now, just discouraging it  from intervening)